docs-src/guides/add-new-blockchain.md

```diff
@@ -75,3 +75,4 @@ Below you can see a table which lists supported blockchains and their provider o
 | Ethereum   | [eip155](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-3.md)    | metamask-like ethereum provider |
 | Filecoin   | [fil](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-23.md)              | [Filecoin Wallet Provider](https://github.com/openworklabs/filecoin-wallet-provider) |
 | EOS        | [eosio](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-7.md)              | [@smontero/eosio-local-provider](https://github.com/sebastianmontero/eosio-local-provider#readme) |
+| Solana     | [solana](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-30.md)              | [@solana/wallet-adapter](https://github.com/solana-labs/wallet-adapter) |
```

Comment:
this whole "supported blockchains" section here is way out of date.  The current list is available here: https://developers.ceramic.network/reference/javascript/blockchain/#currently-supported-blockchains.

Can you update this PR to actually just delete this whole section?

Reply:
done